app/storage/metadata_parser.py

```diff
@@ -131,10 +129,6 @@ def convert_schema_name(self, data, **kwargs):
             logger.info(
                 "Using schema_name claim to specify schema, overriding survey, form_type and region_code"
             )
-        else:
-            data["schema_name"] = get_schema_name_from_census_params(
-                data.get("survey"), data.get("form_type"), data.get("region_code")
-            )
```

Comment:
The equivalent of this will be needed for business surveys - runner expects to have a `schema_name` as this is persisted in the session and used for future loads of the schema.

Reply:
Added the equivalent or business surveys, thanks!